ansible/roles/operator-pipeline/templates/openshift/tasks/trigger-prow-tests.yml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: trigger-prow-tests
+spec:
+  params:
+    - name: pipeline_image
+      description: |
+        The common pipeline image.
+
+    - name: request_url
+      description: |
+        The GitHub issue or pull request URL where we want to add a new
```

Comment:
Please update the description to reflect a context of the task (labels instead of comment)